receiver/couchbasereceiver/testdata/apiresponses/get_clusters_response.json

```diff
@@ -0,0 +1,351 @@
+        },
+        "setAutoCompaction": {
+            "uri": "/controller/setAutoCompaction?uuid=e65468352e51e1f29b3fee22265ba6fa",
+            "validateURI": "/controller/setAutoCompaction?just_validate=1"
+        },
+        "clusterLogsCollection": {
+            "startURI": "/controller/startLogsCollection?uuid=e65468352e51e1f29b3fee22265ba6fa",
+            "cancelURI": "/controller/cancelLogsCollection?uuid=e65468352e51e1f29b3fee22265ba6fa"
+        },
+        "replication": {
+            "createURI": "/controller/createReplication?uuid=e65468352e51e1f29b3fee22265ba6fa",
+            "validateURI": "/controller/createReplication?just_validate=1"
+        }
+    },
+    "rebalanceStatus": "none",
+    "rebalanceProgressUri": "/pools/default/rebalanceProgress",
+    "stopRebalanceUri": "/controller/stopRebalance?uuid=e65468352e51e1f29b3fee22265ba6fa",
+    "nodeStatusesUri": "/nodeStatuses",
+    "maxBucketCount": 30,
+    "maxCollectionCount": 1200,
+    "maxScopeCount": 1200,
+    "autoCompactionSettings": {
+        "parallelDBAndViewCompaction": false,
+        "databaseFragmentationThreshold": {
+            "percentage": 30,
+            "size": "undefined"
+        },
+        "viewFragmentationThreshold": {
+            "percentage": 30,
+            "size": "undefined"
+        },
+        "indexCompactionMode": "circular",
+        "indexCircularCompaction": {
+            "daysOfWeek": "Sunday,Monday,Tuesday,Wednesday,Thursday,Friday,Saturday",
+            "interval": {
+                "fromHour": 0,
+                "toHour": 0,
+                "fromMinute": 0,
+                "toMinute": 0,
+                "abortOutside": false
+            }
+        },
+        "indexFragmentationThreshold": {
+            "percentage": 30
+        }
+    },
+    "tasks": {
+        "uri": "/pools/default/tasks?v=130197264"
+    },
+    "counters": {
+        "rebalance_success": 2,
+        "rebalance_start": 2
+    },
+    "indexStatusURI": "/indexStatus?v=459813",
+    "serverGroupsUri": "/pools/default/serverGroups?v=14667634",
+    "clusterName": "otelc",
+    "clusterEncryptionLevel": "none",
+    "balanced": true,
+    "checkPermissionsURI": "/pools/default/checkPermissions",
+    "memoryQuota": 512,
+    "indexMemoryQuota": 256,
+    "ftsMemoryQuota": 256,
+    "cbasMemoryQuota": 1199,
+    "eventingMemoryQuota": 256,
+    "storageTotals": {
+        "ram": {
+            "total": 12409712640,
+            "quotaTotal": 1610612736,
+            "quotaUsed": 1610612736,
+            "used": 12031537152,
+            "usedByData": 123608048,
+            "quotaUsedPerNode": 536870912,
+            "quotaTotalPerNode": 536870912
+        },
+        "hdd": {
+            "total": 439074951168,
+            "quotaTotal": 439074951168,
+            "used": 30735246579,
+            "usedByData": 61994688,
+            "free": 408339704589
+        }
+    }
+}
```

Comment:
```suggestion
}

```

Reply:
Saw this on a few of the JSON files I'll update.